src/components/search/fallback-disclaimer-message.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <FallbackDisclaimer class="x-fallback-message x-message">
```

Comment:
I think there is an if statement missing here.

Reply:
@herrardo this is a funny one because it only happens in prod and renders the component even though the underlaying component (`FallbackDisclamer`) has an `v-if` for `fromNoResultsWithFilters` and that variable is `fromNoResultsWithFilters` 🤷‍♂️ anyways adding the same `v-if` to `FallbackDisclamerMessage` seems to work well.

![image](https://github.com/empathyco/x-archetype/assets/7124620/f127fddb-a077-437a-9617-d227e60f8705)

Reply:
I keep seeing flaky behaviour summarized in the video:

- We shouldn't see spellcheck and fallback filters message at the same time
- In the step two is working well but step three is not


https://github.com/empathyco/x-archetype/assets/4663897/1cf76d9f-348f-4ad2-9b6c-9c2a2ef8e93a